src/js/App/Header/Tools.js

```diff
@@ -32,20 +47,37 @@ const Tools = () => {
     });
   }, []);
 
+  {
+    /* list out the items for the settings menu */
+  }
+  const settingsMenuDropdownItems = [
+    {
+      title: 'Settings',
+      onClick: () => (window.location = settingsPath),
+    },
+    {
+      title: betaSwitcherTitle,
+      onClick: () => (window.location = switchRelease(window.insights.chrome.isBeta(), window.location.pathname)),
+    },
+  ];
+
   {
     /* button that should redirect a user to RBAC with an account */
   }
   const SettingsButton = () => (
-    <Button
-      variant="plain"
-      aria-label="Go to settings"
-      ouiaId="chrome-settings"
-      className="ins-c-toolbar__button-settings"
-      href={settingsPath}
-      component="a"
-    >
-      <CogIcon />
-    </Button>
+    <ToolbarToggle
```

Comment:
Can we add an ouiaId here? It can be the same as before `chrome-settings`.

Reply:
just added it, sorry :-)

Reply:
Thanks!